.env

```diff
@@ -31,4 +31,5 @@ DATABASE_URL="mysql://db_user:db_password@127.0.0.1:3306/slub"
 SLACK_TOKEN=YOUR-SLACK-BOT-TOKEN
 SUPPORTED_CI_CHECKS=ci_check_1,ci_check_2
 BOT_USER_ID=my_bot_user_id
+WARN_LOC_LIMIT=500
```

Comment:
how about: PR_SIZE_LIMIT ? Since "PR" is already a domain concept I find it clearer.

Reply:
500 is too low, I'd put it to 800 to start, what do you think ? (I know most PR are around 750 loC :/)

Reply:
Alright!